blog/_posts/2017-08-02-bear-hugs-from-200ok.md

```diff
@@ -0,0 +1,92 @@
+
+As the deadline drew closer, we were so fixated by coala, that we didn’t even think of getting involved in any other project. It was coala all the way for us <3
+
+When we got a mail from the organising team in April, we really couldn’t believe it. The subsequent interview with Anika and Ramon and finally, the much awaited results, were actually a dream come true! 
+
+We were one of the four teams from India which made it to the top 10, making it the only country apart from Germany to achieve such a feat. Wow! It was indeed a moment of pride and joy for us to be a part of this wonderful initiative.
+
+### Five Reasons Why coala
+
+- Our project, coala is a code analysis tool. So, developers at coala write bears to help you check for bad coding practices, styles etc. in different languages. And who doesn't want to help people write maintainable code?
+- coala can be used by anyone around the world, whether they have contributed to the project are not.
+- Bear writing at coala is of basically two types:
+ — Native Bears
+ — Linter Bears
+A lot of programming languages already have linters implemented, so if a project uses a language that does not already have a linter bear we do not need to implement it on our own. Linter bears are python wrappers (since coala is in python) around existing code linting tools available in other languages.
+- coala has a very robust CI pipeline including checks for commit and code quality (of course!), Travis, Appveyor, Circle CI and codecov.
+- Oh, and something that we have experienced for the first time at an open source org — at coala, the more code reviews you do — the faster your code gets reviewed by the community (they have a lot of developers and so a lot of PRs!) So, we need to perform code reviews too :D
+
+### The Crew
+As is true with any show, it is the people behind the scenes who actually make it all happen. For us, our wonderful coaches, Vidur, Tushar and Suryansh, our beloved supervisor Mayar (we like to call her our team buddy) and our project mentor, Udayan, are the people guiding us through this summer. We are incredibly blessed and humbled to have such a supportive team, which is the actual reason for getting a 200 OK this summer! We can’t thank you guys enough! <3
+
+<img src="/img/blog/2017/200okcrew.jpg" align="center">
+<div align="center" class="image-credits">(clockwise order from top left) Udayan, Prachi and Ipshita, Mayar, Vidur, Suryansh, Tushar</div>
```

Comment:
please add a credit for the images here too; since it's a collage, you could write something like: `(Photo collage: Team 200 OK)` — it should be enough.